src/knowrob.pl

```diff
@@ -27,6 +27,8 @@
 knowrob_load_neem(NEEM_id) :-
 	% assign DB collection prefix
 	set_setting(mng_client:collection_prefix, NEEM_id),
+	mng_get_db(_DB, CollectionName, 'tf'),
+	mongolog_update_fluent_options('tf_raw',[collection(CollectionName)]),
```

Comment:
I do not like that this is done centrally here with a hardcoded atom "tf_raw". this name should only appear in the tf module